VirtualH89/Src/MMS316IntrCtrlr.h

```diff
@@ -0,0 +1,43 @@
+///
+/// \name MMS316IntrCtrlr.h
+///
+/// Implementation of the H89 interrupt controller circuits, when augmented by the
+/// MMS 77316 floppy disk controller. Used only if the MMS77316 is installed.
+///
+/// \date Feb 8, 2016
+/// \author Douglas Miller
+///
+
+#ifndef MMS316INTRCTRLR_H_
+#define MMS316INTRCTRLR_H_
+
+#include "config.h"
+#include "h89Types.h"
+#include "InterruptController.h"
```

Comment:
Same here with the includes, both of the classes can be forward declared, and the includes moved to the CPP file. Consider this a general comment for any other include files.